src/contour/Config.cpp

```diff
@@ -155,6 +155,11 @@ namespace
 #endif
     }
 
+    std::string replaceCommentPlaceholder(std::string const& docString)
+    {
+        return std::regex_replace(docString, std::regex { "\\{comment\\}" }, "#");
+    }
+
```

Comment:
We might want to customize it for different writers in the future, so we should make it writer type dependent, something like 
```suggestion
    template <typename Writer=YAMLConfigWriter>
    std::string replaceCommentPlaceholder(std::string const& docString)
    {
        return std::regex_replace(docString, std::regex { "\\{comment\\}" }, Writer::comment);
    }

```

or make it a function of Writer type

Reply:
I think I'll leave potential config API/implementation improvements to you :)